lib/economic/endpoint.rb

```diff
@@ -8,6 +8,10 @@ class Economic::Endpoint
   def_delegator "client.globals", :log_level, :log_level=
   def_delegator "client.globals", :log, :log=
 
+  def initialize(app_identifier = nil)
```

Comment:
Will do. My preference would be to add the link to the blog post in the commit message and add a note to the README - sound good?